.azure-pipelines/templates/maven-preview-stage/copy-and-build.yml

```diff
@@ -0,0 +1,18 @@
+steps: 
+- task: CopyFiles@2
+  inputs:
+    SourceFolder: '$(Agent.TempDirectory)'
+    Contents: '**'
+    TargetFolder: '$(System.DefaultWorkingDirectory)/_msgraph-sdk-java-core build and package/drop/'
+
+- task: Gradle@2
+  inputs:
+    gradleWrapperFile: '$(System.DefaultWorkingDirectory)/_msgraph-sdk-java-core build and package/drop/gradlew'
```

Comment:
the paths probably need to be updated, regardless use a variable to avoid duplication

Reply:
Variable DROP_PATH =  $(System.DefaultWorkingDirectory)/_msgraph-sdk-java-core build and package/drop